open-metadata-implementation/access-services/subject-area/subject-area-api/pom.xml

```diff
@@ -6,7 +6,6 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <groupId>org.odpi.egeria</groupId>
```

Comment:
I think it best o leave this line in - so it is explicit unless @planetf1 thinks it is ok.

Reply:
This is a line and so is explicitly specified in the parent tag;)
<img width="350" alt="Screenshot 2020-09-01 at 11 15 56" src="https://user-images.githubusercontent.com/35165540/91825476-02acd080-ec45-11ea-9a5e-b2ee7490b1f1.png">

Reply:
I agree with @immmus - there is no need to have this groupId in (though should be harmless). Our other poms typically do not include it.